Integrations/integration-PagerDutyV2.yml

```diff
@@ -27,6 +27,11 @@ configuration:
   defaultvalue: ""
   type: 13
   required: false
+- display: Initial Fetch Interval(In minutes)
```

Comment:
Please add a few words explaining that this is only used for the first fetch.
I know it says "Initial" there, but customers can miss it (like i did on first read) and think that they are setting the actual fetch interval.